dev/scripts/src/alluxio.org/build-distribution/cmd/generate-release-tarballs.go

```diff
@@ -56,7 +56,8 @@ func Release(args []string) error {
 
 func generateTarballs() error {
 	fmt.Printf("Generating tarball %v\n", fmt.Sprintf("alluxio-%v-bin.tar.gz", versionMarker))
-	if err := generateTarball(strings.Split(hadoopDistributionsFlag, ",")); err != nil {
+	// Do not skip UI and Helm
+	if err := generateTarball(strings.Split(hadoopDistributionsFlag, ","), false, false); err != nil {
```

Comment:
why add these 2 args to generateTarball and is `skipUIFlag` flag ignored now? i would remove the extra args and rely on flags alone. also, `skipHelm` should be `true` here anyways?

Reply:
I thought the `skipUIFlag` only exists for `generate-tarball single` as defined here?
https://github.com/Alluxio/alluxio/blob/5dc73825dc6a2558b0e479b31e2230460e2b4a5d/dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go#L42

I changed the signature of `generateTarball()` passing in the flags just trying to make `skipHelm=false`. AFAIK `skipUIFlag` and `skipHelm` flags don't exist in `generate-tarballs release`. See example below:

```
$ bash generate-tarballs release -skipUI=true
flag provided but not defined: -skipUI
Usage of release:
  -debug
        whether to run this tool in debug mode to generate additional console output
  -hadoop-distributions string
        a comma-separated list of hadoop distributions to generate Alluxio clients for (default "default,hadoop-2.2,hadoop-2.3,hadoop-2.4,hadoop-2.5,hadoop-2.6,hadoop-2.7,hadoop-2.8,hadoop-2.9,hadoop-3.0,hadoop-3.1,hadoop-3.2")
  -mvn-args string
        a comma-separated list of additional Maven arguments to build with, e.g. -mvn-args "-Pspark,-Dhadoop.version=2.2.0"
  -target string
        an optional target name for the generated tarball. The default is alluxio-${VERSION}.tar.gz. The string "${VERSION}" will be substituted with the built version. Note that trailing ".tar.gz" will be stripped to determine the name for the Root directory of the generated tarball (default "alluxio-${VERSION}-bin.tar.gz")
  -ufs-modules string
        a comma-separated list of ufs modules to compile into the distribution tarball(s). Specify 'all' to build all ufs modules. Supported ufs modules: [ufs-hadoop-2.2,ufs-hadoop-2.3,ufs-hadoop-2.4,ufs-hadoop-2.5,ufs-hadoop-2.6,ufs-hadoop-2.7,ufs-hadoop-2.8,ufs-hadoop-3.0,ufs-hadoop-3.1,ufs-hadoop-3.2] (default "ufs-hadoop-2.2,ufs-hadoop-2.7")
exit status 2
```